modules/logging-release-notes-5-9-0.adoc

```diff
@@ -0,0 +1,79 @@
+
+[id="logging-release-notes-5-9-0-enhancements"]
+== Enhancements
+
+[id="logging-release-notes-5-9-0-log-collection"]
+=== Log Collection
+* This enhancement adds the ability to refine the process of log collection by using a workload's metadata and to `drop` or `prune` logs based on their content. Additionally, it allows the collection of infrastructure logs, such as journal or container logs, and audit logs, such as `kube api` or `ovn` logs, to only collect individual sources. (link:https://issues.redhat.com/browse/LOG-2155[LOG-2155])
+
+* This enhancement introduces a new type of remote log receiver, the Syslog Receiver. You can configure it to expose a port over a network, allowing external systems to send Syslog logs using compatible tools such as Rsyslog. (link:https://issues.redhat.com/browse/LOG-3527[LOG-3527])
+
+* With this update, the `ClusterLogForwarder` API now supports log forwarding to Azure Monitor Logs, giving users better monitoring abilities. This feature helps users to maintain optimal system performance and streamline the log analysis processes in Azure Monitor, which speeds up issue resolution and improves operational efficiency. (link:https://issues.redhat.com/browse/LOG-4605[LOG-4605])
+
+* This enhancement improves collector resource utilization by deploying collectors as a deployment with two replicas. It happens when the only input source defined in the `ClusterLogForwarder` is a receiver input instead of using a daemon set on all nodes. Additionally, collectors deployed in this manner do not mount the host file system. To use this enhancement, you need to annotate the `ClusterLogForwarder` with `logging.openshift.io/dev-preview-enable-collector-as-deployment`. (link:https://issues.redhat.com/browse/LOG-4779[LOG-4779])
+
+* This enhancement introduces the capability for custom tenant configuration across all supported outputs, facilitating the organization of log records in a logical manner. However, it does not permit custom tenant configuration for {logging} managed storage. (link:https://issues.redhat.com/browse/LOG-4843[LOG-4843])
+
+* With this update, the `ClusterLogForwarder` that specifies an application input with one or more infrastructure namespaces like `default`, `openshift*`, or `kube*`, now requires a service account with the `collect-infrastructure-logs` role. (link:https://issues.redhat.com/browse/LOG-4943[LOG-4943])
+
+* This enhancement introduces the capability for tuning some output settings, such as compression, retry duration, and max payloads, to match the characteristics of the receiver. Additionally, this feature includes a delivery mode to allow administrators to choose between throughput and log durability. For example, `AtLeastOnce` configures minimal disk buffering of collected logs so that the collector can deliver those logs after a restart. (link:https://issues.redhat.com/browse/LOG-5026[LOG-5026])
```

Comment:
```suggestion
* This enhancement introduces the capability for tuning some output settings, such as compression, retry duration, and maximum payloads, to match the characteristics of the receiver. Additionally, this feature includes a delivery mode to allow administrators to choose between throughput and log durability. For example, the `AtLeastOnce` option configures minimal disk buffering of collected logs so that the collector can deliver those logs after a restart. (link:https://issues.redhat.com/browse/LOG-5026[LOG-5026])
```